app/decorators/category_decorator.rb

```diff
@@ -20,10 +20,19 @@ def canonical_url(posts)
   end
 
   def page_title(posts)
+    if site.category_title_format.present?
+      begin
+        title = site.category_title_format % {category_name: name}
+      rescue KeyError
```

Comment:
Ah right. the validation is already set.
In this case, is `rescue` unnecessary?